csrc/ops/normalization.h

```diff
@@ -212,6 +212,17 @@ NVF_API BackwardRMSNormResult rms_norm_backward(
     TensorView* weight,
     const std::vector<bool>& output_mask);
 
+// From thunder generated python definiton
+// root-mean-square is saved instead of reciprocal rms.
+// Only quires one inner reduction instead of two.
+NVF_API BackwardRMSNormResult thunder_rms_norm_backward(
```

Comment:
nitpick: is it necessary to have this in our c++ API, would a util function in the test suffice?

i.e. is there a use-case where we would want to use this instead of the vanilla `rms_norm`?

Reply:
We don't have a user case. The main reason I put it side-by-side with the original `rms_norm_backward()` is to reminder us there are different flavors of definiton for backward pass and our cpp default version is different from the one generated by thunder. I believe we should revise our default version to keep it same as the thunder version. It influences our cpp benchmarks, but not urgent since we are moving to python benchmarks.

Reply:
sounds fair. Maybe a comment on this stating that this is the new definition that we are transitioning.

> reminder us there are different flavors of definiton for backward pass

After the transition, we can move the old definition in cpp test files instead, just so we have some better coverage on benchmarking. I'm not a big fan of duplicating the c++ API, because that doesn't add any value IMHO.